src/bezier.cpp

```diff
@@ -572,20 +508,15 @@ Curve Curve::offsetCurve(const Curve& curve, double offset, unsigned order)
   return fromPolyline(offset_polyline, order ? order : curve.order() + 1);
 }
 
-Curve Curve::joinCurves(const Curve& curve1, const Curve& curve2, unsigned int order)
+Curve Curve::joinCurves(const Curve& curve1, const Curve& curve2, unsigned order)
```

Comment:
The only time "true" join can be done if you have two Bezier curves of same order N, and they are parametrically continuous to Nth order, and then the resulting curve will be of same order N.

In all of the other cases, you can't join two Bezier Curves. This is the reason why we fit a new curve over their concatenated polylines. You can specify your own wanted order, or in case you don't know, we try to chose some "one heuristic fits all" order. If you choose too small, you get degenerate curve, too high, approximation is too hard to calculate. Even if you choose perfectly, the input curves might be degenerate

Reply:
Okay. Thank you for the explaining.